cms/djangoapps/contentstore/tests/test_tasks.py

```diff
@@ -62,23 +63,15 @@ def test_exception(self, mock_export):  # pylint: disable=unused-argument
         result = export_olx.delay(self.user.id, key, 'en')
         self._assert_failed(result, json.dumps({'raw_error_msg': 'Boom!'}))
 
-    def test_invalid_user_id(self):
+    @mock.patch('cms.djangoapps.contentstore.tasks.User.objects.get', side_effect=User.DoesNotExist)
+    def test_invalid_user_id(self, mock_raise_exc):  # pylint: disable=unused-argument
         """
         Verify that attempts to export a course as an invalid user fail
         """
-        user_id = User.objects.order_by('-id').first().pk + 100
+        user = UserFactory(id=User.objects.order_by('-id').first().pk + 100)
         key = str(self.course.location.course_key)
-        result = export_olx.delay(user_id, key, 'en')
-        self._assert_failed(result, f'Unknown User ID: {user_id}')
-
-    def test_non_course_author(self):
```

Comment:
Was the removal of this test accidental?

Reply:
Oh yes.

Reply:
It was accidental and reverted now.